src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="/src/images/imac.jpeg"
+        alt="Product"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__subtitle">Product code: 195434</p>
+
+      <div class="card__response">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__text">Reviews:5</div>
+      </div>
+
+      <div class="card__price">
+        <p class="left">Price:</p>
+        <p class="right">$2,199</p>
+      </div>
+      <a
+        href="#"
+        class="card__buy card__buy--active"
+        data-qa="hover"
+      >
+        BUY
+      </a>
```

Comment:
According to BEM methodology, the `--active` modifier in the `card__buy--active` class should indicate some state of the `card__buy` block. However, it seems like this class is not changing any state of the `card__buy` block. If you're using it just to style the `card__buy` block, you should remove the `--active` modifier and add these styles to the `card__buy` block itself.